lib/class-wp-rest-plugins-controller.php

```diff
@@ -194,7 +249,16 @@ public function get_item_schema() {
 	}
 
 	public function get_collection_params() {
-		return array();
+		$params = parent::get_collection_params();
+
+		$params['offset'] = array(
```

Comment:
They are being "inherited" (not really but kind of) from `WP_REST_Controller` via `$params = parent::get_collection_params()`.  My interpretation of the API led me to believe that is how these pagination arguments were meant to be brought in.  Should we explicitly add them here or maybe add a comment above the call to the controller class explaining that `page / per_page` come from there?